### ISSUE TITLE
not all systems showing up in menu after loading roms for many systems

### ISSUE THREAD

**Issue by pepsi0457** [libretro/Lakka-LibreELEC#600]
I am using the latest version of lakka 2.1.1 i believe it is, and i am using it on a quad core 1ghz laptop
After loading roms and bios' for multiple systems, not all of the roms show up for the playstation, one rom i loaded doesnt show up in its own list for the psp, and i have no menu for the 2 roms that i loaded for Sega CD. i can go into and start them physically for the psx and the sega cd roms, I have not tried for the psp, also as i was thinking of it i am also missing a list for the gamecube as i have one rom for that that was scanned, that i havent tried to run yet. Has anyone else run into this issue? could it be possible that i have to have them in a correctly labeled folder inside the roms directory, or should they be zipped, unzipped, 7z or rar? any help would be appreciated. thank you

**Comment by RobLoach** [comment]
Provide examples of ones that don't show up. Compare against libretro database.

**Comment by pepsi0457** [comment]
Sure... There is no menu for sega cd or psp... On psp I loaded god of war chains of olympus, and sega cd I loaded night trap and sonic cd... The psx menu showed up with only disk 2 of rebel assault 2 when I loaded metal gear solid, rebel assault, and both disks of rebel assault 2. Also missing dreamcast menu for zelda wind waker. All those are in roms directory and were added/scanned. How do I compare with the database?
